src/resolvelib/resolvers.py

```diff
@@ -224,6 +226,7 @@ def _backtrack(self):
 
             # Retract the last candidate pin, and create a new (b).
             name, candidate = self._states.pop().mapping.popitem()
+            self._r.backtrack(candidate)
```

Comment:
Would `backtracking` be more consistent?

Reply:
It would be correct; given that the method is called `backtracking`. ;P

Reply:
Point taken :-)

Reply:
Sigh. Simple examples don't backtrack. Thanks for spotting this :-)

Reply:
I’ve been thinking about it for a while, this could be the tipping point I start adding type hints to the code base.